runtime/Cargo.toml

```diff
@@ -5,7 +5,7 @@ license = "AGPL-3.0"
 homepage = "https://alan-lang.org"
 documentation = "https://docs.alan-lang.org"
 repository = "https://github.com/alantech/alan"
-version = "0.1.2"
+version = "0.1.3"
```

Comment:
why do we need to bump the runtime?

Reply:
I made a change to the runtime yesterday but I forgot to version bump and publish it.